src/components/Search.js

```diff
@@ -0,0 +1,42 @@
+import React, { useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import SearchIcon from '../assets/images/search-line.png';
+import { searchCity } from '../redux/teams/teams';
+
+function Search() {
+  const [search, setSearch] = useState('');
+  const dispatch = useDispatch();
+  const cities = useSelector((state) => state.cities.cities);
+
+  const handleChange = (event) => {
+    event.preventDefault();
+    setSearch(event.target.value);
+  };
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    const searchCities = [];
+    const city = cities.find((city) => (city.city.toLowerCase()) === search.toLowerCase());
+    searchCities.push(city);
+    if (city) {
+      dispatch(searchCity(searchCities));
+    } else {
+      setSearch('');
+    }
+  };
+
+  return (
+    <form onSubmit={handleSubmit}>
+      <input
+        type="text"
+        placeholder="Search by City"
+        value={search}
+        onChange={handleChange}
+      />
+      <button type="submit" className="navigation-icons">
+        <img src={SearchIcon} alt="Search icon" />
+      </button>
+    </form>
+  );
+}
```

Comment:
You did a great job working on your search bar! 💯 However, there are a few problems related to it that must be solved in order to approve this project! 

- [ ] When I search for a city and want to go back and see all available cities, I'm not able to, even if I search with an empty input. Please consider fixing this! 🔧 

- [ ] It would be great if you handle the case in which the user doesn't enter the exact name of the city, but a close one, like `"Atlant"` or `"Atla"` you can use the String method "Includes" to achieve this! 🤓 🔧 

- [ ] **[OPTIONAL]**: I would suggest, that if you want to maximize the user experience, you dispatch this action on every change, so the user can see a live filter feature! 🤓 🔧